### PR TITLE
DOC-744, DOC-553: Improved data type definitions, Added descriptions from docs

### PR DESCRIPTION
**Pull request by elenaili**
- base58 patterns
- add security to /transactions/sign/{signerAddress}
- delete 403 response from /utils/seed/, /debug/validate
- extend description for /debug/stateHash/{height}
- add description from docs

### CODE REVIEW

**Review by phearnot — APPROVED** [review]

**Comment by msmolyakov** [comment]
https://github.com/wavesplatform/Waves/blob/a9ea6c8e14abe0009c6d1d5e2b14ccbff3307b54/node/src/main/resources/swagger-ui/swagger.json#L527
Not all data, but for array of specified keys.

**Comment by msmolyakov** [comment]
https://github.com/wavesplatform/Waves/blob/a9ea6c8e14abe0009c6d1d5e2b14ccbff3307b54/node/src/main/resources/swagger-ui/swagger.json#L3955

Forever or for a specific time?

**Review by msmolyakov — CHANGES_REQUESTED** [review]

File: node/src/main/resources/swagger-ui/swagger.json [hunk 1]
```diff
@@ -3314,14 +3138,14 @@
         "tags": [
           "assets"
         ],
-        "summary": "Information about the multiple assets",
-        "description": "Provides detailed information about the given assets",
+        "summary": "Information about multiple assets",
+        "description": "Get detailed information about given assets. See [fields descriptions](https://docs.waves.tech/en/blockchain/token/#custom-token-parameters)",
```
Comment (msmolyakov) [inline node/src/main/resources/swagger-ui/swagger.json:3142]: What's the limit for assets count?

**Review by msmolyakov — CHANGES_REQUESTED** [review]

**Review by msmolyakov — APPROVED** [review]

File: node/src/main/resources/swagger-ui/swagger.json (hunk 1 above)
Comment (elenaili) [reply node/src/main/resources/swagger-ui/swagger.json:3142]: limits are not set yet

**Pull request merged**